src/sites_toolbox.py

```diff
@@ -434,19 +434,16 @@ def sites_toolbox_page():
                         unsafe_allow_html=True,
                     )
                 else:
-                    site_group = get_site_group_by_name(
-                        session=session, site_group_name=site_group_name
-                    )
 
                     user = create_user(
                         session=session,
                         email=email,
-                        site_group_name=site_group.site_group_name,
+                        site_group_name=site_group_name,
                     )
 
```

Comment:
@peterdudfield, this commit should fix the confusion between `site_group` and `site_group_name`

Reply:
nice, and I can see you have removed a call to get site group